components/activity_log_modal/components/activity_log.test.tsx

```diff
@@ -32,18 +33,19 @@ describe('components/activity_log_modal/ActivityLog', () => {
     });
 
     test('submitRevoke is called correctly', () => {
-        const wrapper = shallow(
+        const wrapper: ShallowWrapper<any, any, ActivityLog> = shallow(
```

Comment:
You can pass a type parameter into `shallow` to simplify the typing
```suggestion
        const wrapper = shallow<ActivityLog>(
```